### PR TITLE
Cleanup of pubsub_gapic.yaml

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
* Switching non-retry timeout to 60 seconds for consistency with the
  retry timeout
* Re-ordering and reformatting to reduce the edit distance with the
  config generator

### CODE REVIEW

File: google/pubsub/v1/pubsub_gapic.yaml [hunk 1]
```diff
@@ -249,68 +249,71 @@ interfaces:
     field_name_patterns:
       topic: topic
     sample_code_init_fields:
-      - messages[0].data
+    - messages[0].data
```
Comment (michaelbausor) [inline google/pubsub/v1/pubsub_gapic.yaml]: We could add a default initial value here. Or could do it in a followup PR

Comment (garrettjonesgoogle) [inline google/pubsub/v1/pubsub_gapic.yaml]: I vote for following PR. I'm doing some cleanup in preparation for mixin support.

Comment (michaelbausor) [inline google/pubsub/v1/pubsub_gapic.yaml]: OK. in that case, was the change from 30000 to 60000 for timeout intentional?

Comment (michaelbausor) [inline google/pubsub/v1/pubsub_gapic.yaml]: Oops just saw your comment, nevermind

**Comment by michaelbausor** [comment]
LGTM

**Pull request merged**